lib/sensors/i2c_mux.hpp

```diff
@@ -0,0 +1,136 @@
+#pragma once
+
+#include "i2c_sensors.hpp"
+
+#include <array>
+#include <cstdint>
+#include <memory>
+#include <optional>
+
+#include <core/logger.hpp>
+#include <core/types.hpp>
+#include <io/i2c.hpp>
+
+namespace hyped::sensors {
+
+constexpr std::uint8_t kDefaultMuxAddress = 0x70;
+constexpr std::uint8_t kMaxNumChannels    = 8;
```

Comment:
Given that this is in `hyped::sensors`, maybe `kMaxNumMuxChannels` for clarity?

Reply:
It just seems like this name is maybe a bit too general at his scope?

Reply:
i agree, missed this